src/main/java/htsjdk/samtools/SAMFileHeader.java

```diff
@@ -253,7 +255,12 @@ public SortOrder getSortOrder() {
         if (so == null || so.equals("unknown")) {
```

Comment:
Should probably get rid of the `so.equals("unknown")` here since that's now a valid value?